docs/UserGuide.md

```diff
@@ -131,6 +134,12 @@ Below are some examples (by the courtesy of Collate's User Guide):
 
 You can use `Start Tags` to mark the start of your contribution. The author specified in the `Start Tags` will be recognized by RepoSense as the author for all lines between a `Start Tag` and the next `End Tag`. If RepoSense cannot find a matching `End Tag` for a `Start Tag` in the same file, it will assume that all lines between the `Start Tag` to the end of the file is authored by the author specified in the `Start Tag`.
 
+#### Standalone Configuration
+Although management of authors can be done in `config.csv`, RepoSense also supports a decentralized way to manage authors per repository.
```

Comment:
Perhaps we can have the separate documents for the repository owners to explain on how to set up the standalone config? 

Here we can simplify it to:
```
Instead of self-configuring all the authors' information, you can have the repository owners include a 
`Standalone Configuration` to maintain their own author's details. For more information, checkout 
[Setup Standalone Configuration].
```